opentech/public/news/views.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.syndication.views import Feed
+
+from opentech.public.news.models import NewsPage, NewsType
+
+
+class NewsFeed(Feed):
+    link = "https://www.opentech.fund/news/"
```

Comment:
Can we get this programatically? If not I should make it a setting.

Reply:
We can get `NewsIndex` page in method `def link(self, obj):` and remove link attribute.